microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md

```diff
@@ -53,6 +54,32 @@ To use the newer version of the browser you must create a new environment and We
 
 Because a user data folder can only be used by one browser process at a time, if you want to use the same user data folder in the WebViews using the new version of the browser, you must close the environment and WebViews that are using the older version of the browser first. Or simply prompt the user to restart the app.
 
+#### CompareBrowserVersions 
+
+Compare browser versions correctly to determine which version is newer, older or same.
+
+> public static int [CompareBrowserVersions](#comparebrowserversions)(string version1, string version2)
+
+Returns -1, 0 or 1 depending on whether version1 is less than, equal to or greater than version2, respectively.
```

Comment:
```suggestion
Returns -1, 0 or 1 depending on whether the first version is less than, equal to or greater than the second version being compared.
```

Reply:
I believe the version1 and version2 here refer to the parameter names in the API signature.

Reply:
Yeah, I think spelling it out (the first version) helps with clarity.